app/src/main/java/com/babylon/wallet/android/presentation/wallet/WalletViewModel.kt

```diff
@@ -178,7 +178,7 @@ data class WalletUiState(
     private val accountsWithResources: List<AccountWithAssets>? = null,
     private val loading: Boolean = true,
     private val refreshing: Boolean = false,
-    private val accountsNeedSecurityPrompt: List<AccountWithSecurityPrompt> = emptyList(),
+    private val entitiesWithSecurityPrompt: List<EntityWithSecurityPrompt> = emptyList(),
```

Comment:
Can I suggest 2 private fields in state like:
1. entitiesWithSecurityPrompt
2. backupWarningVisible

and a backing field like 
```
val isSettingsWarningVisible: Boolean
    get() = entitiesWithSecurityPrompt.any { it.entity is Network.Persona } || backupWarningVisible
```
just to make the logic a bit simpler and to have only one value exposed from state?